src/components/interfaces/MOBILE_API.xml

```diff
@@ -788,6 +795,7 @@
             <description> Optional hone number of intended location / establishment (if applicable) for SendLocation.</description>
         </element>
         
+        <element name="timeToDestination"/>
```

Comment:
@IGapchuk please add description for this parameter:
```
<description>Text of time to reach destination to be displayed on screen.</description>
```

Reply:
Done. Description was added.